src/CommunityVoices/App/Website/Presentation/Module/Form/Quote.xslt

```diff
@@ -59,6 +59,21 @@
                             </div>
                         </div>
                     </xsl:if>
+                    <xsl:if test="domain/repeatedQuote = 'true'">
+                        <div class="card" style="margin-bottom: 16px;">
+                            <div class="card-body">
+                                <h1 class="h4 mb-4 font-weight-normal" style="margin-bottom: 0.5rem !important;">
+                                  Success!
+                                </h1>
+                                <ul style="margin-bottom: 0.5rem;">
+                                    <xsl:comment>
+                                     Don't know how to make the text below appear as a bullet in the same way as errors are displayed.
+                                    </xsl:comment>
+                                    Your quote has been succesfully uploaded.
```

Comment:
This is a `ul`, so you add a bullet point just by wrapping it in `li`.  The template code that the errors box calls:
```
    <xsl:template match="domain/errors/*|domain/upload/errors/*">
        <li style="margin-bottom: 0px;"><xsl:value-of select="." /></li>
    </xsl:template>
```

Reply:
Will something like this work? (wondering if logic is wrong, or wrong syntax since this isn't working). If not, happy to use <li> tags too. Just figured that this is cleaner

<xsl:template match="domain/errors/*|domain/upload/errors/*|domain/repeatedQuote/*">
        <li style="margin-bottom: 0px;"><xsl:value-of select="." /></li>
</xsl:template>

Reply:
You would therefore need to format your `repeatedQuote` XSL like `errors` where it is a list of error messages.  Right now, `repeatedQuote` is not a list of messages, but rather only true or false.  I would check up on how `errors` is defined and where it comes from in the API.